client/my-sites/email/form/mailboxes/types.ts

```diff
@@ -1,5 +1,15 @@
 import { v4 as uuid_v4 } from 'uuid';
-import { FIELD_DOMAIN, FIELD_UUID } from 'calypso/my-sites/email/form/mailboxes/constants';
+import {
+	FIELD_ALTERNATIVE_EMAIL,
+	FIELD_DOMAIN,
+	FIELD_FIRSTNAME,
+	FIELD_IS_ADMIN,
```

Comment:
I'm thinking that this field should be deprecated, I don't think we use it anymore. But we can defer this to when the component is ready and shipped.